### PR TITLE
Style reports submitted by non competition delegate differently

### PR DESCRIPTION
**Pull request by jfly**
# Before

image

# After

image

### CODE REVIEW

File: WcaOnRails/app/helpers/competitions_helper.rb [hunk 1]
```diff
@@ -16,16 +16,72 @@ def competition_message_for_user(competition, user)
...
+    date ? (date.to_date - competition.end_date).to_i : nil
+  end
+
+  def announced_content(competition)
+    days_announced = days_before_competition(competition.announced_at, competition)
+    days_announced ? "#{pluralize(days_announced, "day")} before" : ""
+  end
+
+  def announced_class(competition)
+    days_announced = days_before_competition(competition.announced_at, competition)
```
Comment (pedrosino) [inline WcaOnRails/app/helpers/competitions_helper.rb]: This seems a repeat of line 28. Maybe you could move it to a separate method/variable?

File: WcaOnRails/app/helpers/competitions_helper.rb [hunk 2]
```diff
@@ -16,16 +16,72 @@ def competition_message_for_user(competition, user)
...
+  private def report_and_results_days_to_class(days)
     level = [Competition::REPORT_AND_RESULTS_DAYS_OK, Competition::REPORT_AND_RESULTS_DAYS_WARNING, Competition::REPORT_AND_RESULTS_DAYS_DANGER].select { |d| days > d }.count
     ["alert-green", "alert-success", "alert-orange", "alert-danger"][level]
   end
 
+  def report_content(competition)
+    days_report = days_after_competition(competition.delegate_report.posted_at, competition)
+    if days_report
+      submitted_by_competition_delegate = competition.delegates.include?(competition.delegate_report.posted_by_user)
+      submitted_by_competition_delegate ? "#{pluralize(days_report, "day")} after" : "submitted by else"
```
Comment (pedrosino) [inline WcaOnRails/app/helpers/competitions_helper.rb]: I'm not a native speaker, so feel free to disconsider this if it doesn't make sense.
"Submitted by else" sounds a little fun...maybe "submitted by other"?

Comment (jfly) [inline WcaOnRails/app/helpers/competitions_helper.rb]: I agree the "submitted by else" sounds weird, that's just what Ilkyoo told me to do. I've change it to "submitted by other".

File: WcaOnRails/app/helpers/competitions_helper.rb (hunk 1 above)
Comment (jfly) [inline WcaOnRails/app/helpers/competitions_helper.rb]: Sure thing. How does it look now?

**Comment by pedrosino** [comment]
LGTM

**Comment by Claster** [comment]
OOC, how is it possible for a non-comp delegate to submit a report?

**Comment by pedrosino** [comment]
Board/results/software team members may "submit" the report if the delegate was removed, for example.

**Pull request merged**